pybossa/cache/projects.py

```diff
@@ -81,8 +82,22 @@ def browse_tasks(project_id, args, filter_user_prefs=False, user_id=None):
             MAX(finish_time) as ft FROM task_run
             WHERE project_id=:project_id GROUP BY task_id) AS log_counts
             ON task.id=log_counts.task_id
-            WHERE task.project_id=:project_id""" + filters + \
+            WHERE task.state='ongoing'
+            AND task.project_id=:project_id""" + filters + \
             " ORDER BY %s" % (args.get('order_by') or 'id ASC')
+    else:
+        sql = """
```

Comment:
Instead of adding if-else in sql, we can consider adding the additional constraint to the function that's generates where clauses - https://github.com/bloomberg/pybossa/blob/cafc4f40bd9009e59442eace3bdcdc0c90d681c5/pybossa/cache/task_browse_helpers.py#L92